apps/frontend/src/libs/components/landing/landing.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./styles.module.css";
+
+const Landing = (): React.JSX.Element => {
+	return (
+		<main className={styles["landing"]}>
+			<div className={styles["landing-header"]}>
+				<h1 className={styles["landing-title"]}>
+					Discover Your Next Adventure
+				</h1>
+
+				<h4 className={styles["landing-description"]}>
```

Comment:
It's just a paragraph from seo perspective. Tags value from seo perspective, and the actual style name on the design may not always match.
```suggestion
				<p className={styles["landing-description"]}>
```